flyway/src/main/scala/com/avast/sst/flyway/FlywayModule.scala

```diff
@@ -0,0 +1,40 @@
+package com.avast.sst.flyway
+
+import cats.effect.Sync
+import org.flywaydb.core.Flyway
+
+object FlywayModule {
+
+  @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+  def make[F[_]: Sync](config: FlywayConfig): F[Flyway] = {
+    Sync[F].delay {
+      val builder = Flyway
+        .configure
+        .dataSource(config.url, config.username, config.password)
```

Comment:
shouldn't we prefer `dataSource​(javax.sql.DataSource dataSource)` (to `dataSource​(java.lang.String url, java.lang.String user, java.lang.String password)`)?

we could then use `Transactor.kernel` as the `javax.sql.DataSource` instance

I think it's generally a good practice to use objects instead of string configuration like usernames, passwords, paths to files as strings and so on

Reply:
OK, let's do `DataSource`.